### ISSUE TITLE
Forwarding encrypted mail doesn't quote message body

### ISSUE THREAD

**Issue by karottenbaum** [k9mail/k-9#3223]
### Expected behavior
When forwarding PGP-encrypted mail, the decrypted message body is included into the forwarding message.

### Actual behavior
The message body is empty except for 
```
-------- Ursprüngliche Nachricht --------
Von: Sender <a@example.com>
Gesendet: 26. Februar 2018 10:00:00 MEZ
An: Recipient <b@example.com>
Betreff: Subject
```
This also happens when sharing or replying to the message.

### Steps to reproduce
1. Be sent encrypted mail
2. Attempt to forward, share or reply

### Environment
K-9 Mail version: 5.403

OpenKeychain version: 4.9 (49001)

Android version: 8.1.0

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

**Comment by Valodim** [comment]
This should be fixed in version 5.501. Please update K-9 on your device, feel free to reopen if you find this issue persists :)

**Comment by karottenbaum** [comment]
Thanks, it's fixed :)
